### ISSUE TITLE
connecting to database with pool doesnt close the connection

### ISSUE THREAD

**Issue by Xcorp24** [hgourvest/node-firebird#240]
I have problem with node-firebird. It is not closing connection at all. Im using db.detach() and closing pools but the connection is still open. I am using firebird monitor app to see how many clients are connected and you can see in the picture there is more than 200 clients

Connection is not closed only if i use POOL. When I use ATTACH its closing connection If I console.log(db) after db.detach() i still see in db object _isDetach: false

sample code:

```
var pool = Firebird.pool(1, options);
pool.get(function (err, db) {

     // error handler
    if (err) console.log(err);
    db.query(tovaryDataQuery, function (err, result) {
    if (err) console.log(err);
    
        res.json(result)
        db.detach();
    })

});
// Destroy pool
pool.destroy();
```
